data/styles/vehicle/include/Icons.mapcss

```diff
@@ -1052,34 +1052,53 @@ area|z16-[amenity=water_point],
 {icon-image: drinking-water-m.svg; icon-min-distance: 20;}
 
 /* 7.1 Parking */
-
-node|z15-16[amenity=parking],
-area|z15-16[amenity=parking]
-{icon-image: parking-m.svg;icon-min-distance: 3;z-index: 20000;}
-node|z15-16[amenity=parking][access=permissive],
-node|z15-16[amenity=parking][access=private]
-{icon-image: zero-icon.svg; icon-min-distance: 3;z-index: 20000;}
-
+node|z15-[amenity=parking],
+area|z15-[amenity=parking]
+{icon-image: parking-m.svg;icon-min-distance: 10;z-index: 20000;font-size: 12.5;}
 node|z17[amenity=parking],
 area|z17[amenity=parking]
-{icon-image: parking-m.svg;icon-min-distance: 15;}
-node|z17[amenity=parking][access=permissive],
-node|z17[amenity=parking][access=private]
-{icon-image: zero-icon.svg;z-index: 6;icon-min-distance: 7;}
-
-node|z18[amenity=parking],
-area|z18[amenity=parking]
-{icon-image: parking-m.svg;icon-min-distance: 10;}
-node|z18[amenity=parking][access=permissive],
-node|z18[amenity=parking][access=private]
-{icon-image: zero-icon.svg;}
-
-node|z19-[amenity=parking],
-area|z19-[amenity=parking]
-{icon-image: parking-m.svg;icon-min-distance: 6;}
+{icon-min-distance: 15;font-size: 13.75;}
+node|z18-[amenity=parking],
+area|z18-[amenity=parking]
+{font-size: 14.5;}
+
+node|z15-[amenity=parking][access=permissive],
+area|z15-[amenity=parking][access=permissive],
+node|z15-[amenity=parking][access=private],
+area|z15-[amenity=parking][access=private],
+node|z15-[amenity=parking][parking=street_side],
+area|z15-[amenity=parking][parking=street_side],
+node|z15-[amenity=parking][parking=lane],
+area|z15-[amenity=parking][parking=lane]
+{icon-image: ;}
+
+node|z15-[amenity=parking][fee?],
+area|z15-[amenity=parking][fee?]
+{icon-image: parking_pay-m.svg;icon-min-distance: 5;font-size: 12.5;}
```

Comment:
its enough to override the icon name only, everything else will come from declarations above